Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/groovy
+
+// Copyright (C) 2019 Luxoft Sweden AB
+
+def image
+
+node("Sashko") {
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage("Build Docker container") {
+        image = docker.build("component", "-f Dockerfile .")
+    }
+
+    image.inside {
+        stage("Download and Install SDK") {
+            sh '''
+            wget https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY_rpi-qtauto/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/pelux-glibc-x86_64-core-image-pelux-qtauto-neptune-dev-cortexa7t2hf-neon-vfpv4-toolchain-*.sh
```

Comment:
why *.sh since you explicitly states the filename below?

Reply:
Due to @dad5a409cd8193942f9bc7068cd394d0d948a8c0 filename changes constantly.

Reply:
Yes I know, but then why can you explicitly name the shell-script?
`pelux-glibc-x86_64-core-image-pelux-qtauto-neptune-dev-cortexa7t2hf-neon-vfpv4-toolchain-master.sh
`
Will that name be constant?